src/viam/sdk/components/board.hpp

```diff
@@ -44,6 +45,15 @@ class Board : public Component {
         std::unordered_map<std::string, digital_value> digital_interrupt_values;
     };
 
+    /// @struct tick
+    /// A board digital interrupt that contains high/low value and the time the digital interrupt
+    /// occured.
+    struct tick {
+        std::string pin_name;
+        bool high;
+        uint64_t time;
```

Comment:
- Slight preference for putting the bool last here.
- Consider using std::chrono for `time`